util/travis.sh

```diff
@@ -1,9 +1,10 @@
 #!/bin/bash
 
-# make sure pull requests include a changelog
```

Comment:
Since the comment changes are unrelated to your other PR changes, could you please revert this file to master?